.github/workflows/ci-nix.yml

```diff
@@ -0,0 +1,38 @@
+name: CI - Nix
+on:
+  push:
+    branches: [main]
+    paths:
+      - '.github/workflows/ci-dev.yml'
+      - '*.nix'
+      - 'flake.lock'
+      - '.ruby-version'
+      - 'dprint.json'
+  pull_request:
+    paths:
+      - '.github/workflows/ci-dev.yml'
+      - '*.nix'
+      - 'flake.lock'
+      - '.ruby-version'
+      - 'dprint.json'
+  schedule:
+    # About JST 8:00
+    - cron: '0 17 * * *'
+  workflow_dispatch:
+
+jobs:
+  tasks:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+      - uses: DeterminateSystems/nix-installer-action@v2
+      - uses: DeterminateSystems/magic-nix-cache-action@v2
```

Comment:
https://github.com/kachick/nix-headbump/commit/ac9ec25ead9463440b38af2a002653aebeb91018 Which is better 👀

Reply:
Or https://github.com/deemp/cache-nix-too

Reply:
Much fast! https://github.com/kachick/eqq/actions/runs/5470886853/jobs/9961455751?pr=120

No concern except cache key name does not have any specifal prefix 🤔 https://github.com/kachick/eqq/actions/caches